trivy-operator.yaml

```diff
@@ -0,0 +1,96 @@
+package:
+  name: trivy-operator
+  version: "0.25.0"
+  epoch: 0
+  description: "Kubernetes-native security toolkit that finds and reports vulnerabilities and misconfigurations"
+  copyright:
+    - license: Apache-2.0
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aquasecurity/trivy-operator
+      tag: v${{package.version}}
+      expected-commit: 0816542808e81dc318a34a1ce8a5cd004c59dfdd
+
+  - uses: go/bump
+    with:
+      deps: |-
+        github.com/containerd/containerd@v1.7.27
+        github.com/containerd/containerd/v2@v2.0.4
+        github.com/golang-jwt/jwt/v4@v4.5.2
+        github.com/golang-jwt/jwt/v5@v5.2.2
+        golang.org/x/oauth2@v0.27.0
+        helm.sh/helm/v3@v3.17.3
+        golang.org/x/net@v0.38.0
+
+  - uses: go/build
+    with:
+      packages: "./cmd/trivy-operator"
+      output: trivy-operator
+      ldflags: |-
+        -X main.version=${{package.version}}
+        -X main.commit=$(git rev-parse HEAD)
+        -X main.date=$(date -u -d "@$SOURCE_DATE_EPOCH" +"%Y-%m-%dT%H:%M:%SZ")
+
+subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/usr/local/bin
+          ln -sf /usr/bin/trivy-operator ${{targets.contextdir}}/usr/local/bin
+    description: Compat package for ${{package.name}}
+    test:
+      pipeline:
+        - runs: |
+            stat /usr/local/bin/trivy-operator
+
+update:
+  enabled: true
+  github:
+    identifier: aquasecurity/trivy-operator
+    strip-prefix: v
+
+test:
+  environment:
+    environment:
+      OPERATOR_NAMESPACE: "trivy-system"
+    contents:
+      packages:
+        - ${{package.name}}
+  pipeline:
+    - name: "Basic binary verification"
+      runs: |
+        # Check if binary produces expected startup message (script exits if grep fails)
+        trivy-operator version 2>&1 | grep -q "Starting operator"
+
+        # Verify package version in ldflags
+        if ! strings /usr/bin/trivy-operator | grep -q "version=${{package.version}}"; then
+          echo "Error: Package version ${{package.version}} not found in binary"
+          exit 1
+        fi
+
+        echo "Basic binary verification passed"
```

Comment:
This binary doesn't have any [subcmds](https://github.com/aquasecurity/trivy-operator/blob/main/cmd/trivy-operator/main.go) at all, let's omit this test as we already cover below.
```suggestion
```